### DISCUSSION TITLE
How do I use Dialogic 2.0 in my Godot 4 game?

### DISCUSSION THREAD

**Discussion by nlupugla** [dialogic-godot/dialogic#1423] [Q&A] [answered]
Hi Dialogic community!

I'm working on a game with some visual-novel elements and I want to use Dialogic because it looks like a great dialogue engine. There are some great tutorials and documentation for Dialogic 1.4 but I could not find any tutorials or documentation for Dialogic 2.0. Can anyone here provide some very basic help on how to use Dialogic inside of one of my existing scenes? In Dialogic 1.4, it was as simple as dragging and dropping Dialogic.tscn into my scene. I don't see an equivalent file in the Dialogic 2.0 addons folder. I started trying to hack something together based on some of the examples scenes in the addons folder, but I'm running into errors and it feels like I'm not using Dialogic "as intended."

Any help is appreciated!

**Comment by QuasiStellar** [comment, marked as answer]
`Dialogic.start("res://[path to the timeline]")` to start the dialogue
Make sure to read docs in the Wiki section

**Comment by Jowan-Spooner** [comment]
As @QuasiStellar said the main thing you are probably missing is `Dialogic.start(@timline)`.

Dialogic.start() automatically adds a scene containing all the nodes that are needed to display/execute your dialog, backgrounds, etc (you can change which scene is added in the settings).
The new thing is that in 2.x you can also add these nodes or a scene containing these nodes manually and just start the execution of a timeline with `Dialogic.start_timeline(@timeline)`. However for you I don't think that is necessary. 

There are many other things that work a bit differently then in 1.x. If you have specific questions, feel free to ask. You can also ask on emilios discord.

**Reply by nlupugla** [reply]
Thanks @QuasiStellar and @Jowan-Spooner! I knew the answer would be very simple. I asked around on the Godot forums and Godot discord but didn't get any responses. The wiki section @QuasiStellar pointed me to is very helpful. I spent a lot of time snooping around for a page like that but somehow never thought to look there. I'm not too familiar with github and I'm used to people linking to their documentation from their readme page.

Would it make sense to add a link to the wiki (maybe even the getting started tutorial specifically) to help new comers like me?

**Reply by Jowan-Spooner** [reply]
There is a link to the getting started page on the Home Page of the plugin (in the godot editor). A link in the readme could be a good idea, the main reason it's not there is that 2.x is still in alpha and documentation has not been a priority yet (meaning the beginners guide is kinda outdated/incomplete).
